docs/topics/api/overview.rst

```diff
@@ -35,6 +41,26 @@ There are some common API responses that you can expect to receive at times.
         The site is in maintenance mode at this current time and the operation
         can not be performed.
 
+~~~~~~~~~~~~
+Bad Requests
+~~~~~~~~~~~~
+
+When returning a ``HTTP 400 Bad Request`` response, the API will try to return
+some information about the error(s) in the body of the response, as a JSON
+object. The keys of that object indicate the field(s) that caused an error, and
+for each, a list of messages will be provided (Often only one message will be
+present, but sometimes more). If the error is not attached to a specific field
+the key ``non_field_errors`` will be used instead.
+
+Example:
+
+     .. code-block:: json
+
+         {
+             "message": ["This field is required."],
```

Comment:
Let's name this `username` to make clear that this is the field-key we're talking about.